regression/verilog/expressions/negation1.desc

```diff
@@ -1,9 +1,8 @@
-KNOWNBUG
+CORE broken-smt-backend
```

Comment:
Just curious: which feature are we missing in the SMT encoding for this test?

Reply:
verilog_case_equality